xlive/Blam/Engine/networking/Session/network_session.h

```diff
@@ -680,5 +681,6 @@ e_network_session_class network_squad_session_get_session_class();
 bool network_session_interface_set_local_user_character_type(int32 user_index, e_character_type character_type);
 bool network_session_interface_get_local_user_identifier(int32 user_index, s_player_identifier* out_identifier);
 void network_session_interface_set_local_user_rank(int32 user_index, int8 rank);
+bool __cdecl network_session_get_scenario_id(uint32* out_1, uint32* out_2, uint32* out_3, s_custom_map_id** out_custom_map);
```

Comment:
remove map_repository include from here and have the parameter as `struct s_custom_map_id**`